test/mix/tasks/kitto.new_test.exs

```diff
@@ -0,0 +1,63 @@
+Code.require_file "../../../installer/lib/kitto_new.ex", __DIR__
+defmodule Mix.Tasks.Kitto.NewTest do
+  use ExUnit.Case, async: false
+  import Plug.Test
+  import Kitto.FileAssertionHelper
+  import ExUnit.CaptureIO
+
+  setup do
+    Mix.Task.clear
+    # The shell asks to install npm and mix deps.
+    # We will politely say not.
+    send self(), {:mix_shell_input, :yes?, false}
+    :ok
+  end
+
+  test "returns the current verions when `--version` is provided" do
+    Mix.Tasks.Kitto.New.run(["--version"])
+    kitto_version = "Kitto v#{Mix.Project.config[:version]}"
+    assert_received {:mix_shell, :info, [^kitto_version] }
+  end
+
+  test "fails when invalid application name is provided" do
+    assert_raise Mix.Error, fn ->
+      Mix.Tasks.Kitto.New.run(["@1@#%$#@"])
+    end
+  end
+
+  test "fails when only provindg a switch" do
+    assert_raise Mix.Error, fn ->
+      Mix.Tasks.Kitto.New.run(["-b"])
+    end
+  end
+
+  test "bootstraps generated project" do
+    in_tmp 'bootstrap', fn ->
+      Mix.Tasks.Kitto.New.run(["photo_dashboard"])
+    end
+
+    in_project :photo_dashboard, Path.join(tmp_path(), "bootstrap/photo_dashboard"), fn _ ->
+      Mix.Task.clear
+      Mix.Task.run "compile", ["--no-deps-check"]
+      # Assert that we are installing, assuming this works
+      assert_received {:mix_shell, :info, ["* creating photo_dashboard/config/config.exs"]}
```

Comment:
Why do we assert on previous output `"* creating photo_dashboard/config/config.exs"` right after running `compile`?
Should this be placed after `Mix.Tasks.Kitto.New.run(["photo_dashboard"])`?